dyno-queues-redis/src/main/java/com/netflix/dyno/queues/redis/sharding/RoundRobinStrategy.java

```diff
@@ -0,0 +1,28 @@
+package com.netflix.dyno.queues.redis.sharding;
+
+import com.netflix.dyno.queues.Message;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class RoundRobinStrategy implements ShardingStrategy {
+
+    private final AtomicInteger nextShardIndex = new AtomicInteger(0);
+
+    /**
+     * Get shard based on round robin strategy.
+     * @param allShards
+     * @param message is ignored in round robin strategy
+     * @return
+     */
+    @Override
+    public String getNextShard(List<String> allShards, Message message) {
```

Comment:
Nice. Couple of concerns.
1. If message is ignored, why not expose an overloaded interface without message?
2. This function has side effect of depending on outside, mutable nextShardIndex. Can we remove this, and make it a pure function?